layouts/shortcodes/plantuml.html

```diff
@@ -0,0 +1,2 @@
+<span id="plantuml-{{ .Get "id" }}" style="display:none">{{ .Inner }}</span>
+<img class="plantuml" id="plantuml-{{ .Get "id" }}">
```

Comment:
since the scripts have been moved to be loaded in scripts.html I think it might be worth starting to use a page store in the short code, something like
```suggestion
<img class="plantuml" id="plantuml-{{ .Get "id" }}">
{{ .Page.Store.Set "hasPlantUML" true }}
```

Reply:
Yep that's smart.  I need to add this to my other JS stuff as well to cut down on load times.

The only issue I ran into with this is when a PlantUML diagram is included in the content summary (the little preview snippet that shows on the homepage) it won't render.  I'm able to get around this by adding a conditional to check if the current page is the home page, but I'm open to suggestions if there's a better way.

```
{{ if or (.Page.Store.Get "hasPlantUML") (.IsHome) }}
<script language="javascript" type="text/javascript" src="{{ "js/plantuml-encoder.min.js" | absURL }}"></script>
<script language="javascript" type="text/javascript" src="{{ "js/plantuml-encoder.js" | absURL }}"></script>
{{ end }}
```